tests/simpleKVBC/TesterReplica/strategy/ShufflePreProcessMsgStrategy.hpp

```diff
@@ -11,17 +11,20 @@
 // LICENSE file.
 
 #pragma once
-
+#include "Logger.hpp"
 #include "TesterReplica/strategy/ByzantineStrategy.hpp"
 
 namespace concord::kvbc::strategy {
 class ShufflePrePrepareMsgStrategy : public IByzantineStrategy {
  public:
   std::string getStrategyName() override;
   uint16_t getMessageCode() override;
-  bool changeMessage(std::shared_ptr<MessageBase> &msg) override;
-  ShufflePrePrepareMsgStrategy() = default;
+  bool changeMessage(std::shared_ptr<bftEngine::impl::MessageBase> &msg) override;
+  explicit ShufflePrePrepareMsgStrategy(logging::Logger &logger) : logger_(logger) {}
```

Comment:
please rename the .hpp and .cpp files accordingly : ShufflePrePrepareMsgStrategy.hpp / ShufflePrePrepareMsgStrategy.cpp

Reply:
Ok, I will rename the files.